### PR TITLE
Package update

### PR DESCRIPTION
**Pull request by Izzmo**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "name": "js-marketing-notification",
-  "version": "1.2.1",
+  "version": "1.3.1",
   "description": "A module for displaying tasteful marketing content to users upon first visit to page.",
-  "main": "src/js-marketing-notification.js",
+  "main": "js-marketing-notification.ts",
```
Comment (someyoungideas) [inline package.json:5]: @Izzmo using the `.ts` file will cause any of our GitHub repos to go through local TS compiler as well. Just stating to see if that is something we are wanting for all going forward.

**Review by someyoungideas — APPROVED** [review]
@Izzmo everything looks good just one comment for discussion more than changes needed.

File: package.json (hunk 1 above)
Comment (someyoungideas) [reply package.json:5]: Never mind looks like we were including that already before. So what does this change handle exactly?

Comment (Izzmo) [reply package.json:5]: @someyoungideas this just removes duplicate information from the package, essentially, allowing it to actually be imported from jspm. Without removing some of these fields, it errors out on install.

**Review by samdotme — APPROVED** [review]

**Pull request merged**